### DISCUSSION TITLE
Какой вариант установки вы предпочитаете?

### DISCUSSION THREAD

**Discussion by ilyhalight** [FOSWLY/vot-cli#44] [Polls]
В связи с планами переписать скрипт, решил сделать этот опрос. Проголосуйте за тот вариант, который кажется вам более удобным.

На данный момент рассматриваю сделать установку через Node.js/Bun.sh + возможность отдельно использовать через бинарник (без необходимости ставить Node.js отдельно, но имеет больший размер файла)

**Comment by InfernalWolf666** [comment]
Я слабо понимаю, слишком запутано, и в терменале требует много разрешений, можно ли сделать просто как yt-dl.exe?
там очень удобно
